### ISSUE TITLE
saxs_fitter requires an internet connection

### ISSUE THREAD

**Issue by franke-biosaxs** [AUSAXS/AUSAXS#106]
I was surprised about the file downloads when I ran it the first time and turned off WiFi to see what happens (contrary to #104, 6lyz.pdb is now the latest version from PDB):
```
% bin/saxs_fitter 6lyz.pdb lyzexp.dat
Running AUSAXS v1.0.3
Reading PDB file from "./6lyz.pdb"
	Successfully read 1102 atomic records.
		101 of these are hydration atoms.
Unknown residue: "LYS". Attempting to download specification.
	curl::download: Failed to download files.rcsb.org/ligands/view/LYS.cif
Failed to download files.rcsb.org/ligands/view/LYS.cif
libc++abi: terminating due to uncaught exception of type std::exception: std::exception
zsh: abort      bin/saxs_fitter 6lyz.pdb lyzexp.dat
```
An online requirement was never mentioned anywhere? Consider adding relevant files to the distribution.

**Comment by klytje** [comment]
Good point. I should add this as a requirement, or at least fix my implementation to avoid crashing when not connected to the internet. Strictly speaking these files are not necessary anyway, as they are only used for determining the number of bound hydrogens for each atom. My main priority has been keeping AUSAXS available as an independent, relocatable binary, which would not be possible if these files are required. I'll try to think of something. Thank you.

**Comment by franke-biosaxs** [comment]
> Strictly speaking these files are not necessary anyway, as they are only used for determining the number of bound hydrogens for each atom. 

And how would you do exactly that, determining the number of hydrogens, without those files? Crysol used to have a preprocessed binary lookup table in the binary, but I replaced that with components.cif for transparency. So, I would advise against doing that. However, this is  also why crysol is relatively slow for small proteins: on startup it first reads a local copy of components.cif, and treats it as a database for lookup. The information has to come from somewhere?

**Comment by klytje** [comment]
I think I can get quite far by hardcoding the most common residues into the program, and then downloading the more exotic residue definitions on the fly. Perhaps emitting a suitable warning when the download fails. 

I think most people will be connected to the internet nowadays, so I don't see a huge issue with this approach. Though I think shipping the complete table is also a good option for ATSAS, it'll be a large burden for my more light-weight program.

**Comment by klytje** [comment]
Fixed in #137.
